include/verilated_fst_c.cpp

```diff
@@ -196,6 +198,9 @@ void VerilatedFst::addCallback(VerilatedFstCallback_t initcb, VerilatedFstCallba
 
 void VerilatedFst::dump(vluint64_t timeui) {
     if (!isOpen()) return;
+    assert(m_minNextDumpTime <= timeui);
+    if (   m_minNextDumpTime >  timeui) return;
```

Comment:
Nit: fix spacing

    if (m_minNextDumpTime > timeui) return;

Actually think this goes away with above assert.